tests/python/frontend/mxnet/test_forward.py

```diff
@@ -464,6 +464,12 @@ def verify(data_shape, weight_shape):
     verify((2, 2), (4, 5))
     verify((2, 3, 4), (4, 5))
 
+def test_forward_bilinear_resize():
```

Comment:
I think we can provide one new more unit test with `scale_height` and `scale_width` to cover our implementation.

Reply:
Does CI has the latest mxnet installed?

Reply:
I'll just try and see if it gives me error.

Reply:
@FrozenGene seems https://mxnet.apache.org/api/python/symbol/contrib.html?highlight=bilinearresize#mxnet.symbol.contrib.BilinearResize2D is not giving the correct guidance, it's not related to mxnet version.

Reply:
@FrozenGene So I confirmed that only Mxnet 1.5 fixes the bugs. If CI can update mxnet, we can then have this test.

Reply:
Could you add this in the comment of this issue?